io.photoflare.photoflare.json

```diff
@@ -0,0 +1,63 @@
+{
+  "app-id": "io.photoflare.photoflare",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.13",
```

Comment:
Not a blocker, but have you tried building it using 5.14?

Reply:
I actually didn't know it was out. Will do.